test/Microsoft.ServiceFabric.Services.Remoting.Tests/Microsoft.ServiceFabric.Services.Remoting.Tests.csproj

```diff
@@ -23,6 +23,5 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
     <Compile Remove="**\*.cs" />
     <Compile Include="ServiceRemotingProviderAttributeTest.cs" />
-    <Compile Include="AssemblyAttributes.cs" />
```

Comment:
In projects where AssemblyAttributes.cs was conditionally included for TargetFramework net472, keep the Compile item, but include the AssemblyInfo.cs instead of the AssemblyAttribute.cs